lib/logging/json_formatter.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'json'
+
+class JSONFormatter
```

Comment:
This would need to be `Logging::JSONFormatter` given the directory structure. Maybe just move it up to `lib/json_formatter.rb`

Reply:
I think I understand this, however not sure how it works now then. Is Logging::ClassName just mor explicit to avoid name collisions?

Reply:
How is this different from `Lograge::Formatters::Json`? Would it be worth extending from it? i.e.
``` ruby
class JSONFormatter < Lograge::Formatters::Json
end
```

Reply:
@whereismyjetpack can you also put in a spec test for the logger? It's lacking test coverage.

Reply:
I believe this to be resolved. I don't think my spec file is terribly idiomatic, So i'm happy to go back and make some changes.